services/agents/registry.go

```diff
@@ -1103,6 +1105,36 @@ func (r *Registry) StartPTSummaryAction(ctx context.Context, id, pmmAgentID stri
 	return nil
 }
 
+// StartPTPgSummaryAction starts pt-pg-summary action on the pmm-agent.
+// The pt-pg-summary may require some of the following params: host, port, username, password.
+// The function returns nil if ok, otherwise an error code
```

Comment:
unnecessary comment

Reply:
It depends on the point of view.

Reply:
What the point of view, there are so many functions which returns error. Why to comment it, when you can see it from function signature?  Code will be bloated with these comments.
It will do more more trouble than good. You already seen it now. You had wrong argument name in comment. You can see arguments in function, why to repeat it comment? Comments should just clarify, if something is not clear.

Reply:
I agree with @Dasio